bokehjs/src/coffee/api/figure.coffee

```diff
@@ -41,14 +40,14 @@ define [
         try
           new Date.parse(range.get('start'))
           return Collections("DatetimeAxis")
-        catch
+        catch e
```

Comment:
Oh weird, it compiles for me but I agree this is better. @fpliger this is not an error check. It's the only way I know of to  detect a date string: try to make a date and if it works, it is a date string, and if it fails, it is not.